### ISSUE TITLE
Error: spawn wmic ENOENT windows 11

### ISSUE THREAD

**Issue by aeroniemi** [microsoft/devicescript#662]
WMIC seems to have become an optional feature in the latest builds of windows 11 and is needed by the VSCode extension to "flash firmware", giving a ``Error: spawn wmic ENOENT`` error if it's missing. It'd be worth adding error handling that directs people to docs on how to enable that

**Comment by pelikhan** [comment]
indeed, it is deprecated --> https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/wmic
